### ISSUE TITLE
check-mk-agent check wrong path

### ISSUE THREAD

**Issue by siliconalchemy** [thbe/puppet-rkhunter#14]
check-mk-agent veresion: check-mk-agent-1.2.6p16-3.el7.x86_64
OS: CentOS 7.2

xinetd.d service path: /etc/xinetd.d/check-mk-agent

```
<% if scope.lookupvar('rkhunter::check_mk') then -%>
XINETD_ALLOWED_SVC=/etc/xinetd.d/check_mk
<% end -%>
```

Path perhaps could be parameterized?

**Comment by thbe** [comment]
Will be fixed in 0.8.1

**Comment by thbe** [comment]
0.8.1 was released on the Puppet forge
